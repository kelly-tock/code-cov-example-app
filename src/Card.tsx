import React, { FunctionComponent } from "react";
import clsx from "clsx";
import "./Card.css";

interface CardProps {
  outlined?: boolean;
}

export const Card: FunctionComponent<CardProps> = ({ outlined, children }) => {
  return (
    <div
      className={clsx("Card", {
        "Card--outlined": outlined,
      })}
    >
      {children}
    </div>
  );
};
