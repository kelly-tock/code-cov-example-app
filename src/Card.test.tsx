import React from "react";
import { render, screen } from "@testing-library/react";
import { Card } from "./Card";

test("matches default snap", () => {
  const { container } = render(<Card>test default button</Card>);
  expect(container.firstChild).toMatchSnapshot();
});
