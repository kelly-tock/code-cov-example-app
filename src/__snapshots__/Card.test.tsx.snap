// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches default snap 1`] = `
<div
  class="Card"
>
  test default button
</div>
`;
